### DISCUSSION TITLE
각 페이지 별 템플릿 담당

### DISCUSSION THREAD

**Discussion by ej20001000** [CodeMedics/Harulog-backend#5] [Announcements]
자신이 담당할 페이지 템플릿을 골라주세요. 일단 골라주시고 다 하신 분은 남은 하나 마저 해주세용.

페이지 별로 footer navbar 고려하며 피그마 참고 하시면서 만들어 주시면 됩니다.

**Comment by jeongmino1** [comment]
깔끔합니다
